.circleci/config.yml

```diff
@@ -36,10 +36,105 @@ parameters:
 
 orbs:
   slack: circleci/slack@4.2.1
-  gravitee: gravitee-io/gravitee@dev:1.0.4
+  gravitee: gravitee-io/gravitee@dev:1.0.26
   secrethub: secrethub/cli@1.1.0
   # secrethub: secrethub/cli@1.0.0
 
+jobs:
+
+  package_bundle:
```

Comment:
I don't understand this job?

As we are using `gravitee/d_cockpit_connectors_bundle` from the orb, we don't need it?